RecoMET/METFilters/plugins/EcalDeadCellDeltaRFilter.cc

```diff
@@ -221,71 +205,42 @@ void EcalDeadCellDeltaRFilter::loadJets(const edm::Event &iEvent, const edm::Eve
 //
 EcalDeadCellDeltaRFilter::EcalDeadCellDeltaRFilter(const edm::ParameterSet &iConfig)
     : jetToken_(consumes<edm::View<reco::Jet> >(iConfig.getParameter<edm::InputTag>("jetInputTag"))),
-      jetSelCuts_(iConfig.getParameter<std::vector<double> >("jetSelCuts"))
-
-      ,
-      metToken_(consumes<edm::View<reco::MET> >(iConfig.getParameter<edm::InputTag>("metInputTag")))
-
-      ,
+      jetSelCuts_(iConfig.getParameter<std::vector<double> >("jetSelCuts")),
+      metToken_(consumes<edm::View<reco::MET> >(iConfig.getParameter<edm::InputTag>("metInputTag"))),
       debug_(iConfig.getUntrackedParameter<bool>("debug", false)),
-      printSkimInfo_(iConfig.getUntrackedParameter<bool>("printSkimInfo", false))
-
-      ,
+      printSkimInfo_(iConfig.getUntrackedParameter<bool>("printSkimInfo", false)),
+      ecalChannelStatusToken_(esConsumes<edm::Transition::BeginRun>()),
+      caloGeometryToken_(esConsumes<edm::Transition::BeginRun>()),
+      ecalTrigTowerConstituentsMapToken_(esConsumes<edm::Transition::BeginRun>()),
       maskedEcalChannelStatusThreshold_(iConfig.getParameter<int>("maskedEcalChannelStatusThreshold")),
-      chnStatusToBeEvaluated_(iConfig.getParameter<int>("chnStatusToBeEvaluated"))
-
-      ,
-      makeProfileRoot_(iConfig.getUntrackedParameter<bool>("makeProfileRoot", true)),
-      profileRootName_(iConfig.getUntrackedParameter<std::string>("profileRootName", "EcalDeadCellDeltaRFilter.root"))
-
-      ,
+      chnStatusToBeEvaluated_(iConfig.getParameter<int>("chnStatusToBeEvaluated")),
       isProd_(iConfig.getUntrackedParameter<bool>("isProd")),
-      verbose_(iConfig.getParameter<int>("verbose"))
-
-      ,
+      verbose_(iConfig.getParameter<int>("verbose")),
       doCracks_(iConfig.getUntrackedParameter<bool>("doCracks")),
       cracksHBHEdef_(iConfig.getParameter<std::vector<double> >("cracksHBHEdef")),
-      cracksHEHFdef_(iConfig.getParameter<std::vector<double> >("cracksHEHFdef"))
-
-      ,
-      EcalDeadCellDeltaRFilterInput_(iConfig.getParameter<std::vector<double> >("EcalDeadCellDeltaRFilterInput"))
-
-      ,
+      cracksHEHFdef_(iConfig.getParameter<std::vector<double> >("cracksHEHFdef")),
+      EcalDeadCellDeltaRFilterInput_(iConfig.getParameter<std::vector<double> >("EcalDeadCellDeltaRFilterInput")),
       taggingMode_(iConfig.getParameter<bool>("taggingMode")) {
   produces<int>("deadCellStatus");
   produces<int>("boundaryStatus");
   produces<bool>();
-
-  if (makeProfileRoot_) {
-    profFile = new TFile(profileRootName_.c_str(), "RECREATE");
-    h1_dummy = new TH1F("dummy", "dummy", 500, 0, 500);
-  }
 }
 
-EcalDeadCellDeltaRFilter::~EcalDeadCellDeltaRFilter() {
-  if (makeProfileRoot_) {
-    profFile->cd();
-
-    h1_dummy->Write();
-
-    profFile->Close();
-    delete profFile;
-  }
-}
+EcalDeadCellDeltaRFilter::~EcalDeadCellDeltaRFilter() {}
```

Comment:
I think this should be omitted by using the default destructor
```suggestion
EcalDeadCellDeltaRFilter::~EcalDeadCellDeltaRFilter() = default;
```